internal/compiler-interface/src/main/java/xsbti/AnalysisCallback.java

```diff
@@ -47,4 +47,10 @@
      *       Do not depend on it, please.
      */
     boolean nameHashing();
+
+    /** Called at the end of dependency phase. Can be used e.g. to wait on asynchronous tasks. */
```

Comment:
Yes. I want to implement async calls in my client (since IIRC classDependency and binaryDependency introduce some overhead but I was able to run in parallel to analyzing compilation units).

I fear I will nee to remove it completely or introduce another callback (since I don't see where I can override it in client). @eed3si9n Any pointers?